modules/virt-cloning-a-datavolume.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-cloning-a-datavolume_{context}"]
+= Cloning a DataVolume
```

Comment:
Since this assembly is for smart cloning, should this task be more specific to be for cloning a DataVolume using smart cloning?

If so, you could consider taking the prerequisites section from `virt-understanding-smart-cloning.adoc` and making it a ".Prerequisites" section for this procedure. So, letting users know that if they want to clone a data volume using smart cloning, then they have to make sure that those prereqs are met first.

Reply:
Yeah, this is kind of sticky point and I went back and forth about it. The problem with specifying smart-cloning in the title is that this procedure will create a clone in either case --- with smart-cloning or host-assisted.  But I do think I should move the smart-cloning prereqs to this section.

Reply:
Yeah I see what you mean. As long as it still makes sense, I do like the prereqs paired with that procedure so that users know right there what needs to be done to make it successful.